src/topology/algebra/group.lean

```diff
@@ -1417,27 +1417,15 @@ The infimum of a collection of group topologies is the topology generated by all
 
 The supremum of two group topologies `s` and `t` is the infimum of the family of all group
 topologies contained in the intersection of `s` and `t`. -/
-@[to_additive]
-instance : complete_semilattice_Inf (group_topology α) :=
-{ Inf_le := λ S a haS, to_topological_space_le.1 $ Inf_le ⟨a, haS, rfl⟩,
-  le_Inf :=
-  begin
-    intros S a hab,
-    apply topological_space.complete_lattice.le_Inf,
-    rintros _ ⟨b, hbS, rfl⟩,
-    exact hab b hbS,
-  end,
-  ..group_topology.has_Inf,
-  ..group_topology.partial_order }
-
 @[to_additive]
 instance : complete_lattice (group_topology α) :=
-{ inf := (⊓),
-  top := ⊤,
-  bot := ⊥,
-  ..group_topology.bounded_order,
+{ ..group_topology.bounded_order,
   ..group_topology.semilattice_inf,
-  ..complete_lattice_of_complete_semilattice_Inf _ }
+  ..group_topology.partial_order,
+  ..complete_lattice_of_Inf (group_topology α) $ λ S, ⟨
```

Comment:
I don't think so. These lemmas aren't def-eq even if they're very similar, because one is about all topological spaces, while the other is about all group topologies.